### ISSUE TITLE
Suggestion: Fairy Rings?

### ISSUE THREAD

**Issue by Jin-Jiyunsun** [MJHylkema/spirit-tree-map#6]
Would it be possible to do something similar for fairy rings?

**Comment by MJHylkema** [comment]
Yup, its in the works. Will take a some time though. We have some other interfaces planned first.

**Comment by FeAspers** [comment]
I started making a fairy ring map and was going to post to make a suggestion and then I found this open issue so I would still like to share my map to help give an idea!
![fairyringsteleportmaps](https://github.com/MJHylkema/spirit-tree-map/assets/73009020/da47459b-a026-4cfd-a897-8a90d010d713)

**Comment by Jin-Jiyunsun** [comment]
Been a while, any progress on this? :) looks cool
